src/map/guild.cpp

```diff
@@ -1492,7 +1492,10 @@ int guild_skillupack(int guild_id,uint16 skill_id,uint32 account_id) {
 
 void guild_guildaura_refresh(struct map_session_data *sd, uint16 skill_id, uint16 skill_lv) {
 	struct skill_unit_group* group = NULL;
-	sc_type type = status_skill2sc(skill_id);
+	sc_type type = skill_get_sc(skill_id);
+
+	if (type == SC_NONE)
+		return;
```

Comment:
Below there is code that takes into account if the SC is already running.
Can the group deletion be moved into sc_end?